Python/Product/PythonTools/PythonTools/Environments/AddCondaEnvironmentControl.xaml

```diff
@@ -0,0 +1,405 @@
+﻿<UserControl x:Class="Microsoft.PythonTools.Environments.AddCondaEnvironmentControl"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008" 
+             xmlns:l="clr-namespace:Microsoft.PythonTools.Environments"
+             xmlns:wpf="clr-namespace:Microsoft.VisualStudioTools.Wpf"
+             xmlns:ptwpf="clr-namespace:Microsoft.PythonTools.Wpf"
+             xmlns:catalog="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.ImageCatalog"
+             xmlns:imaging="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.Imaging"
+             xmlns:common="clr-namespace:Microsoft.PythonTools;assembly=Microsoft.PythonTools.Common"
+             mc:Ignorable="d" 
+             d:DesignHeight="450" d:DesignWidth="730">
+    <UserControl.CommandBindings>
+        <CommandBinding
+            Command="{x:Static wpf:Commands.BrowseOpenFile}"
+            CanExecute="Browse_CanExecute"
+            Executed="Browse_Executed" />
+        <CommandBinding
+            Command="{x:Static l:AddCondaEnvironmentControl.BrowsePackages}"
+            CanExecute="BrowsePackages_CanExecute"
+            Executed="BrowsePackages_Executed" />
+    </UserControl.CommandBindings>
+    <UserControl.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ResourceDictionary Source="pack://application:,,,/Microsoft.PythonTools;component/PythonTools/Wpf/ModernStyles.xaml" />
+                <ResourceDictionary>
+                    <wpf:Lambda x:Key="BoolToVisible">(bool b) => b ? Visibility.Visible : Visibility.Hidden</wpf:Lambda>
+                    <wpf:Lambda x:Key="BoolToHidden">(bool b) => b ? Visibility.Hidden : Visibility.Visible</wpf:Lambda>
```

Comment:
In RTVS, we had a set of predefined lambda converters: https://github.com/Microsoft/RTVS/blob/master/src/R/Wpf/Impl/Converters.cs
It helps to avoid defining these common things in every control or view

Reply:
Thanks. I'm tired of defining them everywhere. I'm integrating those now.